### DISCUSSION TITLE
React modal with react-transition-group

### DISCUSSION THREAD

**Discussion by nandita1** [reactjs/react-modal#1007] [Q&A]
Hello, I was trying to add `CSSTransition` with react-modal. But was unable to do. Can anyone help me with this?

**Comment by diasbruno** [comment]
@nandita1 You can use [Inline Styles](http://reactcommunity.org/react-modal/styles/) or [Classes ](http://reactcommunity.org/react-modal/styles/classes/).

You can find an example of transition using classes here [Transitions](http://reactcommunity.org/react-modal/styles/transitions/).
